File: tests/snapshots/vec.rs
// Vec<T> — a growable list

fn vbr_main() -> Result<(), String> {
    let mut nums: Vec<i64> = Vec::new();
    nums.push(10);
    nums.push(20);
    nums.push(30);
    println!("count = {}", nums.len());
    let mut total: i64 = 0;
    for n in &nums {
        total = total + *n;
    }
    println!("total = {}", total);
    Ok(())
}

fn main() {
    if let Err(error) = vbr_main() {
        eprintln!("Error: {error}");
        std::process::exit(1);
    }
}

File: tests/snapshots/vec.py
# Vec<T> — a growable list

def _vb(x):
    if isinstance(x, bool):
        return "true" if x else "false"
    if isinstance(x, float) and x.is_integer():
        return str(int(x))
    return str(x)

def main():
    nums: list[int] = []
    nums.append(10)
    nums.append(20)
    nums.append(30)
    print(f"count = {_vb(len(nums))}")
    total: int = 0
    for n in nums:
        total = total + n
    print(f"total = {_vb(total)}")


if __name__ == "__main__":
    main()
